_data/misc.yaml

```diff
@@ -0,0 +1,45 @@
+releases:
+  core:
+    latest:
+      1-11: 1.11.0-RC1
+      1-10: 1.10.2
+      1-9: 1.9.3
+    all:
+      - "1.1"
+      - "1.2"
+      - "1.3"
+      - "1.4"
+      - "1.5"
+      - "1.6"
+      - "1.7"
+      - "1.8"
+      - "1.9"
+      - "1.10"
+      - "1.11"
+  plugins:
+    foreman_ansible:
+      - "0.x"
```

Comment:
+1 Discovery does a lot of minor revisions, for example the first official release for 1.11 RC will be 5.0.1. We should be able to list them all there. Hopefully Foreman shows the incoming plugin updates to users too.